mesonbuild/interpreter.py

```diff
@@ -2232,6 +2233,14 @@ def add_devenv_method(self, args: T.Union[str, list, dict, EnvironmentVariablesH
             env = EnvironmentVariablesHolder(env)
         self.build.devenv.append(env.held_object)
 
+    @noPosargs
+    @permittedKwargs({'unbuffered'})
+    def get_command(self, args, kwargs):
```

Comment:
Missing @FeatureNew('meson.get_command', '0.58.0')

Reply:
I will add the requested parts if we are clear on whether we want to support unbuffered or not.
> * reference manual doc
> * release notes snippet
> * unit test

For the final pull request I will squash the branch, this is a small, on commit change.